kubectl-volsync/cmd/migration_create.go

```diff
@@ -36,18 +48,29 @@ var migrationCreateCmd = &cobra.Command{
 	and it sets up an associated ReplicationDestination that will be configured
 	to accept incoming transfers via rsync over ssh.
 	`)),
-	RunE: doMigrationCreate,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		r := &migrationCreate{
+			Command: *cmd,
+		}
+		return r.Run()
+	},
 	Args: validateMigrationCreate,
 }
 
 func init() {
 	migrationCmd.AddCommand(migrationCreateCmd)
 
-	migrationCreateCmd.Flags().String("accessmodes", "", "AccessModes of the PVC to create")
-	migrationCreateCmd.Flags().String("capacity", "", "capacity of the PVC to create")
+	migrationCreateCmd.Flags().String("copymethod", "",
+		"Copy method used to copy the data from source to destination PVC")
+	migrationCreateCmd.Flags().String("accessmodes", "",
+		"accessMode of the PVC to create. viz: ReadWriteOnce, ReadOnlyMany, ReadWriteMany, ReadWriteOncePod")
+	migrationCreateCmd.Flags().String("capacity", "", "size of the PVC to create in Gi ex: 10Gi")
```

Comment:
`access mode and capacity`  should be mandatory args?

If PVC pre-exists then capacity won't be a requirement, since we are using the pre-existing PVC.
Also, the access mode can default to `READWRITEONCE` if not provided

Reply:
good catch. accessMode and capacity are only needed if PVC doesn't exist.

I'm ok w/ defaulting to RWO.

Reply:
sure